run_brer/run_config.py

```diff
@@ -583,6 +583,19 @@ def run(self, tpr_file=None, **kwargs):
             else:
                 assert end_time > start_time
                 if end_time - start_time >= requested_production_time:
+                    state_dir = os.path.dirname(self.state_json)
+                    # The state.json file is overwritten below.
+                    # We retain previous state.json files from
+                    # completed BRER iteration "i" as state_i.json.
+                    # See https://github.com/kassonlab/run_brer/issues/24.
+                    archive_name = 'state_' + str(self.run_data.get('iteration')) + '.json'
+                    prev_iter_state_json = os.path.join(state_dir, archive_name)
+                    if os.path.exists(prev_iter_state_json):
+                        raise RuntimeError(
+                            'If you intended to re-run the phase, please remove ', prev_iter_state_json,
+                            '.', 'See https://github.com/kassonlab/run_brer/issues/24')
```

Comment:
The commas mean this is going to get parsed as separate arguments to `RuntimeError`.
You should be able to just get rid of the commas, but it might improve readability to format the message before the `raise` line. How about this?
```suggestion
                        message = 'If you intended to re-run the phase, please remove '
                        message += str(prev_iter_state_json)
                        message += '. See https://github.com/kassonlab/run_brer/issues/24'
                        raise RuntimeError(message)
```